jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -13,6 +13,7 @@ export IMAGE_OS
 export DEFAULT_HOSTS_MEMORY
 export NUM_NODES
 export UPGRADE_TEST
+export EPHEMERAL_TEST
```

Comment:
it is already in the vars.sh which is sourced in L8. I think this is redundant.

Reply:
I understand, but follow the same pattern like other variables such as UPGRADE_TEST

Reply:
Yeah this section is a bit funky with the exports, but I am not sure if you should fix it here or in an other PR though.

Reply:
Let's tackle this on a follow up PR @smoshiur1237 
/approve